java/spacewalk-java.spec

```diff
@@ -329,10 +240,8 @@ This package contains the code for the Java version of the Spacewalk Web Site.
 %package config
 Summary:        Configuration files for Spacewalk Java
 Group:          Applications/Internet
-%if 0%{?suse_version}
 Requires(post): apache2
```

Comment:
`apache2` doesn't exist in RH/Fedora. It's known as `httpd` there. The SUSE `apache2` package also provides `httpd`, so that name can be used for both.

Reply:
I checked this and it will not work as "nginx" also Provides: httpd .
It would just result in a choise which needs to be solved via project config.

I think it is better to be explicit. I have an idea how to do it.